docs/content/customization/docker/custom-image.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Custom image"
++++
+
+The custom image customization allows the user to specify the OCI image to use for control-plane and worker Machines.
+
+This customization will be available when the
+[provider-specific cluster configuration patch]({{< ref "..">}}) is included in the `ClusterClass`.
+
+## Example
+
+To specify the custom image, use the following configuration:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: <NAME>
+spec:
+  topology:
+    variables:
+      - name: clusterConfig
+        values:
```

Comment:
```suggestion
        value:
```

Reply:
Thanks, its actually `values:` here, but changed fixed it in the other doc.

Reply:
may be I am looking at wrong place. I think https://github.com/kubernetes-sigs/cluster-api/blob/main/api/v1beta1/cluster_types.go#L327 is related to this object?? 🤔

Reply:
🤦 yep you're right! I was looking at a different example in the docs which had `values`